### PR TITLE
Add vert resize feature

### LINKED ISSUE #136: Allow for TinyMCE editor pane to be resized
Currently the editor pane appears at the bottom at a fixed height, or in fullscreen mode it covers the entire preview. There should be a way to resize the pane as desired.

See https://wordpress.org/support/topic/suggested-features-2?replies=4#post-8395773 /cc @pingram3541

### PR DESCRIPTION
**Pull request by pingram3541**
Adds vertical resize capability to post editor pane

Fixes #136

### CODE REVIEW

File: js/customize-post-section.js [hunk 1]
```diff
@@ -306,6 +317,46 @@
...
+                    }
+                } );
+            } );
+
+			$( document ).on( 'mouseup', function() {
+                $( document ).unbind( 'mousemove' );
+                $( '#cpdraghelper' ).remove();
+                editorFrame.css( 'pointer-events', '' );
+            });
+
```
Comment (westonruter) [inline js/customize-post-section.js]: Looks like spaces are being used for indentation when tabs are required.

File: js/customize-post-section.js [hunk 2]
```diff
@@ -306,6 +317,48 @@
 				editor.focus();
 			};
 
+			/**
+             * Vertically Resize Expanded Post Editor leaving nor less than 300px of preview or editor
+             */
+            dragbar.on( 'mousedown', function( mdEvent ) {
+                var wh = window.innerHeight,
```
Comment (valendesigns) [inline js/customize-post-section.js]: I would like to see more clearly named variables. Using acronyms is not obvious for those unfamiliar with what the code does.

File: js/customize-post-section.js [hunk 3]
```diff
@@ -306,6 +317,48 @@
...
+
+				$( document ).on( 'mousemove', function( mmEvent ) {
+					resize = wh - mmEvent.pageY;
+					editorFrame.css( 'pointer-events', 'none' );
+
+					if ( resize < minScroll ) { //Don't scale lower than 300px
+						preview.css( 'bottom', '300px' );
+						editorPane.css( 'height', '300px' );
+						editorFrame.css( 'height', '200px' );
+						dragbar.css( 'bottom', '299px' );
```
Comment (valendesigns) [inline js/customize-post-section.js]: We could likely DRY up this CSS manipulation by creating a function that receives the 3 unique values.

File: js/customize-post-section.js [hunk 4]
```diff
@@ -306,6 +317,48 @@
...
+             */
+            dragbar.on( 'mousedown', function( mdEvent ) {
+                var wh = window.innerHeight,
+                sh = mceTools.outerHeight() + mceToolbar.outerHeight() + mceStatusbar.outerHeight(),
+                bh = 1, //Element border offset
+                minScroll = 300,
+                maxScroll = 300;
+
+                mdEvent.preventDefault();
+                preview.prepend( '<div id="cpdraghelper"></div>' );
```
Comment (valendesigns) [inline js/customize-post-section.js]: Let's not abbreviate the CSS class name, instead use `customize-posts-draghelper`.

File: php/class-wp-customize-posts.php [hunk 5]
```diff
@@ -512,7 +512,7 @@ public function enqueue_editor() {
 	 * Render rich text editor.
 	 */
 	public function render_editor() {
-		echo '<div id="customize-posts-content-editor-pane">';
+		echo '<div class="cp-dragbar"></div><div id="customize-posts-content-editor-pane">';
```
Comment (valendesigns) [inline php/class-wp-customize-posts.php]: Just like above we should not abbreviate the CSS class name, instead use `customize-posts-dragbar`.

File: js/customize-post-section.js (hunk 3 above)
Comment (pingram3541) [inline js/customize-post-section.js]: Question, now that function with arguments is involved, when setting the min/max allowances for the editor's vertical height, I originally set this to 300px for both so at minimum it is never re-sized below the default height and at maximum it always leaves 300px of the preview iframe showing.

However, I found when using this feature myself I preferred having max set to 1 and min to 39.  This allowed for full screen sizing and at minimum the top-most editor tool bar remains visible.  Do you have a preference?  Do you foresee any likelihood of these arguments needing to be user definable?

**Comment by valendesigns** [comment]
This commit https://github.com/xwp/wp-customize-posts/pull/138/commits/4260f27ce4e48962e1c58420bb3792a791963891 removed code that should not have been and is why the tests are failing.

**Comment by valendesigns** [comment]
I'm cleaning up this PR and testing thoroughly, should be ready shortly.

**Comment by valendesigns** [comment]
Closing in favor of #149

**Pull request closed without merging**